velodrome/fetcher/conversion.go

```diff
@@ -0,0 +1,165 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/test-infra/ciongke/github"
```

Comment:
I've been using `github.com/kubernetes/test-infra` because last I checked `go get k8s.io/test-infra/ciongke/stuff` doesn't work. We should probably make those consistent.

I'm also not sure if it's a good idea to use this library here. It's just a really thin wrapper around `go-github` and I didn't intend for it to be used by other projects. Because velodrome is in the same repo I'll hit breaking changes in CI, but it still worries me.